src/transformers/trainer.py

```diff
@@ -3143,7 +3173,10 @@ def training_step(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor,
             with amp.scale_loss(loss, self.optimizer) as scaled_loss:
                 scaled_loss.backward()
         else:
-            self.accelerator.backward(loss)
+            if not is_lomo_optimizer:
+                self.accelerator.backward(loss)
+            else:
+                self.optimizer.optimizer.fused_backward(loss, self._get_learning_rate())
```

Comment:
This makes me believe that we want to do this at the `Accelerate` level, to be quite honest. Simply because `accelerator.backward` handles grad accum dividing and a bunch of gradient scaling. 

Let's discuss this offline for a bit

Reply:
yeah makes sense, happy to upstream that into accelerate to make things cleaner! ok will ping you offline